src/main/java/ch/njol/skript/conditions/CondIsWet.java

```diff
@@ -0,0 +1,45 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import org.bukkit.entity.Entity;
+
+@Name("Is Wet")
+@Description("Checks whether an entity is wet or not (in water, rain or a bubble column).")
+@Examples({"player is wet"})
+@RequiredPlugins("Paper 1.16+")
+@Since("INSERT VERSION")
+
+public class CondIsWet extends PropertyCondition<Entity> {
+	
+	static {
+		if (Skript.methodExists(Entity.class, "isInWaterOrRainOrBubbleColumn"))
+			register(CondIsWet.class, PropertyType.BE, "wet", "entities");
+	}
+
+	@Override
+	public boolean check(Entity entity) { return entity.isInWaterOrRainOrBubbleColumn(); }
```

Comment:
Can you change the formatting on this method and the property name one? Just to be like:
```suggestion
	public boolean check(Entity entity) {
		return entity.isInWaterOrRainOrBubbleColumn();
	}
```
(not sure if spacing / tabs are formatted right in this suggestion as I'm on mobile)

Reply:
I knew someone will comment on this :D I just copied the same template from `CondCanFly` and waited for someone to comment about it lol

Reply:
Why'd you make it like that if you knew it had to be changed?

Reply:
Because it was like this already when I duplicated it but I had a feeling that someone will comment on this didn't know for sure

Reply:
Many of the review comments on your PRs are related to formatting, while you should be able to get how Skript's formatting works from the code conventions, and from other Skript files
https://github.com/SkriptLang/Skript/blob/2497729601fb66e47051e1e12b44b886c6a6a5c3/src/main/java/ch/njol/skript/conditions/CondCanFly.java#L34-L54

Reply:
Weird! I think my IDE shortened it or something my bad. Sure I am getting to used to Skript's format